src/context/AppProvider/AppContext.tsx

```diff
@@ -235,7 +234,75 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     fetchMarketData() // fetch every time assetIds change
     const interval = setInterval(fetchMarketData, 1000 * 60 * 2) // refetch every two minutes
     return () => clearInterval(interval) // clear interval when portfolioAssetIds change
-  }, [dispatch, isPortfolioLoaded, portfolioAssetIds])
+  }, [dispatch, portfolioLoadingState, portfolioAssetIds])
+
+  const toastId = 'accountError'
+
+  const handleAccountErrorToastClose = useCallback(
+    () => toast.isActive(toastId) && toast.close(toastId),
+    [toast],
+  )
+
+  /**
+   * portfolio loading error notification
+   */
+  useEffect(() => {
+    const erroredAccountIds = entries(portfolioLoadingStateGranular).reduce<AccountId[]>(
```

Comment:
Nitpick, could be filtered but potato potahto

Reply:
Disregard me, `.filter().map()` is a double iteration while `reduce()` is only one